rs/crypto/internal/crypto_service_provider/src/signer/tests.rs

```diff
@@ -383,21 +352,6 @@ mod verify_ed25519 {
         );
     }
 
-    #[test]
-    fn should_correctly_verify_batch_with_other_csp() {
```

Comment:
Note to self: Test is only relevant to CSP which doesn't exist anymore (as far as verification is concerned anyway)